pkg/backup/main.go

```diff
@@ -22,7 +22,7 @@ var (
 	log     = logrus.WithFields(logrus.Fields{"pkg": "backup"})
 )
 
-type BackupParameters struct {
```

Comment:
NIT: Retain the previous naming. This won't be used in the restore function

Reply:
Updated. Thank you.